app/(app)/feedback.tsx

```diff
@@ -1,20 +1,144 @@
 import MainContainer from '../../components/container/MainContainer';
+import FeedbackSentModal from '../../components/modals/FeedbackSentModal';
+import { typography } from '../../config/typography';
 
-// TODO: implement feedback page
-/**
- * TASKS:
- * Implement feedback page
- * Should include a reactions component, a textarea and button
- */
+export type TypeRatingStates = {
+  icon: string;
+  text: string;
+  value: number;
+};
+
+const Feedback = () => {
+  const { colors } = useTheme();
+  const router = useRouter();
+  const [showModal, setShowModal] = useState(false);
+  const [_selectedRating, setSelectedRating] = useState(2);
+  const [description, setDescription] = useState('');
+  const ratingStates: Array<TypeRatingStates> = [
+    { icon: '😔', text: 'Bad', value: 0 },
+    { icon: '😐', text: 'Okay', value: 1 },
+    { icon: '😊', text: 'Great', value: 2 },
+  ];
 
-const feedback = () => {
   return (
-    <MainContainer preset="scroll">
-      <StyledText>feedback</StyledText>
+    <MainContainer preset="scroll" safeAreaEdges={['top']}>
+      <FeedbackSentModal showModal={showModal} />
+      <View style={styles.FeedBackBannerParent}>
+        <FeedBackBanner />
+        <Pressable style={styles.headerTitle} onPress={() => router.back()}>
+          <AntDesign name="arrowleft" size={24} color={'white'} />
+          <StyledText style={[{ color: colors.whiteConstant }]} size="lg">
+            Feedback
+          </StyledText>
+        </Pressable>
+      </View>
+      <View style={styles.FeedBackFormContainer}>
+        <StyledText size="lg" font="bold" variant="text" style={[styles.FeedBackFormTitle, { color: colors.primary }]}>
+          Your feedback helps us improve
+        </StyledText>
+        <View style={[styles.FeedBackForm, { backgroundColor: colors.background, borderColor: colors.border }]}>
+          <StyledText style={styles.FeedBackFormLabel} size="md">
+            How is/was the event
+          </StyledText>
+          <View style={styles.FeedBackFormRatingContainer}>
+            {ratingStates.map((rating, index) => {
+              return <FeedBackRatingButton rating={rating} key={index} onPress={setSelectedRating} />;
+            })}
+          </View>
+        </View>
+        <TextInput
+          style={[
+            styles.feedbackInput,
+            { backgroundColor: colors.bg, borderColor: colors.borderColor, color: colors.bgInverse },
+          ]}
+          placeholder="Type message here"
+          placeholderTextColor={colors.placeHolder}
+          value={description}
+          onChangeText={setDescription}
+        />
+        <Pressable
+          style={[styles.pressableSubmit, { backgroundColor: colors.assetAccent }]}
+          onPress={() => {
+            setShowModal(true);
+          }}
+        >
+          <StyledText style={[styles.pressableSubmitText]} font="bold">
+            SUBMIT FEEDBACK
+          </StyledText>
+        </Pressable>
```

Comment:
You can convert this button into a component.
Also the text font is semibold

Reply:
fixed in commit 6eec3b8d02ba41d71546e019f34f0096d80963fa